samples_project/Assets/SampleViewer/Samples/Geocoding/Geocoder.cs

```diff
@@ -35,11 +36,14 @@ public class Geocoder : MonoBehaviour
     public float LocationMarkerScale = 1;
     public GameObject AddressCardTemplate;
     public TextMeshProUGUI InfoField;
+    public Button SearchButton;
 
     private Camera MainCamera;
     private GameObject QueryLocationGO;
     private ArcGISMapComponent arcGISMapComponent;
+    private Animator animator;
     private string ResponseAddress = "";
+    private string textInput;
     private bool ShouldPlaceMarker = false;
     private bool WaitingForResponse = false;
     private float Timer = 0;
```

Comment:
Sort the variables alphabetically please

Reply:
Now that's purely a person choice. Other sample codes do not necessarily have this sorted alphabetically. Alphabetical sort is semantically meaningless.